sveltblazers/Dockerfile

```diff
@@ -1,8 +1,15 @@
 FROM node:20
+
 WORKDIR /app
+
 COPY package*.json ./
+
 RUN npm install
+
 COPY . .
```

Comment:
I was able to get a ~2 second build time down from 18.5 seconds by not copying over the _entire_ directory. Seems like this could be a little faster. Lmk if you want to do this in a subsequent update or would rather fix it in this one.